tests/fixtures/bigquery_fixtures.py

```diff
@@ -0,0 +1,304 @@
+import logging
```

Comment:
I like that the fixtures for BigQuery are separated from the rest. It makes it easier to work with 👍